internal/service/token/token.service.go

```diff
@@ -167,3 +169,39 @@ func (s *serviceImpl) RemoveRefreshTokenCache(refreshToken string) error {
 
 	return nil
 }
+
+func (s *serviceImpl) CreateResetPasswordToken(userId string) (string, error) {
+	resetPasswordToken := s.CreateRefreshToken()
+	tokenCache := &tokenDto.ResetPasswordTokenCache{
+		UserID: userId,
+	}
+	err := s.resetPasswordTokenCache.SetValue(resetPasswordToken, tokenCache, 900)
```

Comment:
should we move this ttl to config?

Reply:
I will retrieve the token from the request , then I will check if the token is exist or not.

Reply:
Yes, i think so

Reply:
I see